### PR TITLE
sign-plugin-docs: information of usage of a new token has been added to the docs

### LINKED ISSUE #328: Update documentation for signing a plugin (https://grafana.com/docs/grafana/latest/developers/plugins/publish-a-plugin/sign-a-plugin/#sign-a-public-plugin)

### PR DESCRIPTION
**Pull request by Ukochka**
The signing package should first try to read a new env variable called GRAFANA_ACCESS_POLICY_TOKEN and use it for signing purposes

We need to change documentation to show how to get this new GRAFANA_ACCESS_POLICY_TOKEN and how to use it

Fixes grafana/plugin-tools#328

**Special notes for your reviewer:**

I have uploaded screenshots to https://admin.grafana.com/upload/static/img/docs/plugins/

### CODE REVIEW

**Comment by Eve832** [comment]
Hey @Ukochka can you please sign the CLA? We can't merge without it.

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md [hunk 1]
```diff
@@ -28,17 +28,26 @@ To make your plugin publicly available outside of your organization, sign your p
...
-## Generate an API key
+## Generate a token
 
-To verify ownership of your plugin, generate an API key that you'll use every time you need to sign a new version of your plugin.
+To verify ownership of your plugin, generate access token that you'll use every time you need to sign a new version of your plugin.
 
 1. [Create a Grafana Cloud account](/signup).
 
-1. Make sure that the first part of the plugin ID matches the slug of your Grafana Cloud account.
+1. Login into your account and navigate to "My Account" -> Security-> Access Policies. And click "Create access pilicy"
```
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:37]:
```suggestion
1. Login into your account and navigate to **My Account > Security > Access Policies**. Click **Create access policy**.
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md [hunk 2]
```diff
@@ -28,17 +28,26 @@ To make your plugin publicly available outside of your organization, sign your p
...
 
-   You can find the plugin ID in the `plugin.json` file inside your plugin directory. For example, if your account slug is `acmecorp`, you need to prefix the plugin ID with `acmecorp-`.
+   Realm: has to be your-org-name (all-stacks)\
```
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:39]:
```suggestion
   Realm: has to be your-org-name (all-stacks)
   
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md [hunk 3]
```diff
@@ -28,17 +28,26 @@ To make your plugin publicly available outside of your organization, sign your p
...
+   Scope: plugins:write
 
-1. [Create a Grafana Cloud API key](/docs/grafana-cloud/reference/create-api-key/) with the **PluginPublisher** role.
+   {{< figure src="/static/img/docs/plugins/create-access-policy.png" class="docs-image--no-shadow" max-width="650px" >}}
+
+1. Once policy created, create a new tocker by clicking "Create token" button.\
```
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:44]:
```suggestion
1. Click **Create token** to create a new token.
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md [hunk 4]
```diff
@@ -28,17 +28,26 @@ To make your plugin publicly available outside of your organization, sign your p
...
+   Token name could be anything\
+   "Expration date" is oprional although we stringly advise to change tokens from time to time for security reason.\
+
+   {{< figure src="/static/img/docs/plugins/create-access-policy-token.png" class="docs-image--no-shadow" max-width="650px" >}}
+
+   Click "Create" and Copy the token
```
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:50]:
```suggestion

   The expiration date field is optional, though you should change tokens periodically for increased security.

   {{< figure src="/static/img/docs/plugins/create-access-policy-token.png" class="docs-image--no-shadow" max-width="650px" >}}

1. Click **Create** and save a copy of the token somewhere secure for future reference. 
```

**Review by Eve832 — CHANGES_REQUESTED** [review]

**Comment by josmperez** [comment]
I concur with Eve's suggestions. Otherwise, looks good.

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md (hunk 1 above)
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md (hunk 2 above)
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md (hunk 3 above)
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md (hunk 4 above)
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md [hunk 5]
```diff
@@ -28,17 +28,26 @@ To make your plugin publicly available outside of your organization, sign your p
 
 If you intend to only use the plugin within your organization, sign it under a _private_ [signature level](#plugin-signature-levels).
 
-## Generate an API key
+## Generate a token
 
-To verify ownership of your plugin, generate an API key that you'll use every time you need to sign a new version of your plugin.
+To verify ownership of your plugin, generate access token that you'll use every time you need to sign a new version of your plugin.
```
Comment (academo) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:33]: `generate access token` -> `generate an access token`

**Comment by tolzhabayev** [comment]
@Ukochka due to how our documentation works ([gh actions](https://github.com/grafana/grafana/blob/main/.github/workflows/publish-technical-documentation-release.yml)), you have to backport this PR to 10.0.x by adding a `backport 10.0.x` label to it. After the merge grafanabot will create a backport PR and link it here. You can approve and merge the backport PR yourself (no need for additional approvals). 

Once merged into main the doc changes will be available at: 
- https://grafana.com/docs/grafana/next/developers/plugins/publish-a-plugin/sign-a-plugin/

Once backport is merged at:
- https://grafana.com/docs/grafana/latest/developers/plugins/publish-a-plugin/sign-a-plugin/

Both might take a while to sync and publish.

**Review by Eve832 — APPROVED** [review]

**Pull request merged**